minifier/README.md

```diff
@@ -0,0 +1,15 @@
+This directory contains tools for the contract size minification.
+Requirements:
+     * cargo install wasm-snip wasm-gc
+     * apt install binaryen wabt
+
+*WARNING*: minification could be rather aggressive, so you *must* test the contract after minificaion.
+Standalone NEAR runtime (https://github.com/nearprotocol/nearcore/tree/master/runtime/near-vm-runner) could be helpful
+here.
+
+Current approach to minification is the following:
+    * snip (i.e. just replace with `unreachable` instruction) few known fat functions from the standard library
+     (such as float formatting and panic related)
```

Comment:
float formatting can actually happen, because we don't prohibit floats. Not that it's needed to be included most of the time.

Reply:
So do we just prohibit the use of float operations?  If so, I'll need to update sdk-as to allow for them.

Reply:
Think float computations can happen in contract, we can just forbid float formatting.